cmd/committee-cli/commands/sync/total_members_attribute.go

```
@@ -0,0 +1,143 @@
+func (s *totalMembersAttributeSubcommand) Name() string { return "total-members-attribute" }
+
+func (s *totalMembersAttributeSubcommand) Help() string {
+	return "reconcile CommitteeBase.TotalMembers against the actual member count in the KV store"
+}
+
+func (s *totalMembersAttributeSubcommand) Run(ctx context.Context, rc commands.RunContext) error {
+	slog.DebugContext(ctx, "starting subcommand", "subcommand", s.Name(), "args", rc.Args)
+
+	fs := flag.NewFlagSet("total-members-attribute", flag.ExitOnError)
+	fs.Usage = func() {
+		fmt.Fprintf(fs.Output(), "usage: committee-cli [--nats-url] sync total-members-attribute [flags]\n\nflags:\n")
```

golangci-lint (2.12.1)

[error] Error return value of `fmt.Fprintf` is not checked

(errcheck)